### DISCUSSION TITLE
Watermeter shows neg rate, but raw is bigger than prev value

### DISCUSSION THREAD

**Discussion by heikohol** [jomjol/AI-on-the-edge-device#3075] [Q&A]
Since a few weeks my Watermeter shows status "Neg rate", but when you look at the message, it says "Neg. Rate - Read: - Raw: 00633.053 - Pre: 630.410", so raw is actually bigger than the prev one.

![image](https://github.com/jomjol/AI-on-the-edge-device/assets/154093458/74dde8f3-4806-4845-bf63-eae5392636d4)
I am running 15.7.0

**Comment by caco3** [comment]
Does it stay like this?
What happens if you set the preValue again?

**Reply by heikohol** [reply]
If I set PreValue close to the actual value, it works for one or two readings but then it happens again like described above.
